apis/nucleus/handler/data-property-handler.js

```diff
@@ -1,11 +1,18 @@
 import extend from 'extend';
-// eslint-disable-next-line import/no-relative-packages
-import isEnabled from '../../../nucleus/src/flags/flags';
-import { findFieldById, useMasterNumberFormat } from './utils/field-helper/field-utils';
+import { findFieldById, initializeField } from './utils/field-helper/field-utils';
 import { INITIAL_SORT_CRITERIAS } from './utils/constants';
 import { notSupportedError } from './utils/hypercube-helper/hypercube-utils';
 
+/**
+ * @class DataPropertyHandler
+ * @description A class to handle data properties for dimensions and measures in a data model.
+ * @export
+ */
 class DataPropertyHandler {
+  /**
+   * Creates an instance of DataPropertyHandler.
+   * @param {object} opts - Options for the handler.
```

Comment:
recommend to also document what possible option that can be in the opts parameter